apps/connect/source/ftrack_connect/__main__.py

```diff
@@ -76,10 +87,30 @@ def main_connect(arguments=None):
 
     namespace = parser.parse_args(arguments)
 
-    ftrack_connect.config.configure_logging(
+    ftrack_connect.utils.log.configure_logging(
         'ftrack_connect', level=loggingLevels[namespace.verbosity]
     )
 
+    # TODO: Discuss if those keys should might be FTRACK_CONNECT_PLUGIN_PATH and
+    #  FTRACK_CONNECT_LAUNCH_PATH even though they are
+    #  not environment variables anymore.
+    default_values = {
+        'type': 'connect_config',
+        'name': 'default-connect-config',
+        'plugin_path': [get_default_plugin_directory(), 'test/path'],
+        'plugin_name': '*',
+        # TODO: double check if {$name of the variable } is something more or less standard or we shouldn't put the $
+        'launch_path': '{$plugin_path}/{$plugin_name}/launch',
```

Comment:
I think it is more standard (compare Docker/Kubernetes YAML config specs) to have variable substitution be on the form ${VAR_NAME}, I am also used to that more overall.